src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in

```diff
@@ -162,6 +165,8 @@ class Predict(BasePredict):
 
         self.validate()
         BasePredict.call_internal_keras(self)
+        if self.mst_key:
+            plpy.execute("DROP VIEW IF EXISTS {}".format(self.model_summary_table))
```

Comment:
It's a bit confusing to read that we are dropping the model summary table. I understand that we overwrite this variable with the temp summary view.
I think it would be better if we use a different variable instead of overwriting the same one.